src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesListFieldArray/testResources.js

```diff
@@ -0,0 +1,113 @@
+const data = {
+  'fields': {
```

Comment:
We should never send `fields`, `input` or `meta` props in directly unless there's a very good reason to. These should already be being handled by the `FieldArray`, so it might work to just remove them directly.